temporal-sdk/src/main/java/io/temporal/internal/client/RootWorkflowClientInvoker.java

```diff
@@ -424,7 +427,8 @@ private void pollWorkflowUpdateHelper(
       PollWorkflowExecutionUpdateRequest request,
       Deadline deadline) {
 
-    Deadline pollTimeoutDeadline = Deadline.after(60L, TimeUnit.SECONDS).minimum(deadline);
+    Deadline pollTimeoutDeadline =
+        Deadline.after(POLL_UPDATE_TIMEOUT_S, TimeUnit.SECONDS).minimum(deadline);
```

Comment:
How long until the server times this out? I just saw Go SDK does this too. I really think we need to be letting server time these out with blank successes like they do our worker polling and history fetching.

Reply:
@mmcshane

Reply:
Doesn't affect this PR, but does effect behavior. To ask another way, what happens when my update takes 90 seconds? Do we error or do we turn to async result fetching?

Reply:
I'll let Matt comment on changing the server behavior. This PR is just to get Go and Java aligned.

>what happens when my update takes 90 seconds? Do we error or do we turn to async result fetching?

I believe we would time out, there is no logic to switch to "async result fetching". Depending on what exactly you mean  by "async result fetching" that might not be possible.

Reply:
That's an application semantic. The underlying transport may or may not support that desire. Best you can do is try to do what is requested and react.